src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,17 @@
+package duke.task;
+
+public class Event extends Task {
+
+    private String description;
+    private String timeslot;
```

Comment:
If "time slot" are two separate words, maybe it is better to use the camel case here "timeSlot".